examples/todomvc/src/containers/FilterLink.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import { setVisibilityFilter } from '../actions'
+import Link from '../components/Link'
+
+const mapStateToProps = (state, ownProps) => ({
+  active: ownProps.filter === state.visibilityFilter
+})
+
+const mapDispatchToProps = (dispatch, ownProps) => ({
+  setFilter: () => {
+    dispatch(setVisibilityFilter(ownProps.filter))
+  }
+})
+
+const FilterLink = connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(Link)
+
+export default FilterLink
```

Comment:
I would prefer doing

```
export default connect(
   mapStateToProps,
   mapDispatchToProps
)(Link)
```

But I'm not sure if this is a convention or not, I just don't see the point in the intermediary variable that is disposable anyway.